server/rosetta/config.go

```diff
@@ -30,17 +32,24 @@ const (
 	DefaultNetwork = "network"
 	// DefaultOffline defines the default offline value
 	DefaultOffline = false
+	// DefaultEnableDefaultFeeSuggest indicates to use fee suggestion
+	DefaultEnableDefaultFeeSuggest = false
```

Comment:
odd to read, we can remove Default prefix and address the changes that @alexanderbez suggested for naming